bftengine/src/bftengine/messages/ReplicaStatusMsg.cpp

```diff
@@ -26,18 +27,26 @@ static uint8_t powersOf2[] = {
     0x80,
 };
 
+static const int kWorkWindowBitMaskSize = (kWorkWindowSize + 7) / 8;
```

Comment:
Nit: variables defined in an anonymous namespace are automatically `static`.
If both `kWorkWindowSize` is a compile-time variable, then `kWorkWindowBitMaskSize` can be a `constexpr`.